tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -1523,6 +1533,11 @@ export const getDisplayedResourcesFromShares = async (page: Page): Promise<strin
 export const getDisplayedResourcesFromTrashbin = async (page: Page): Promise<string[]> => {
   const files = []
   const result = page.locator('[data-test-resource-path]')
+  try {
+    await result.first().waitFor({ timeout: config.minTimeout * 1000 })
+  } catch (err) {
+    console.log('Trashbin is empty')
+  }
```

Comment:
it might not work if user user is member of the project space
<img width="1581" alt="Screenshot 2024-06-12 at 10 37 45" src="https://github.com/owncloud/web/assets/84779829/c793db76-2e30-4d40-a13f-57cab9160c81">

Reply:
yeah, it's true. does the test fail?
I think we have separate step to get the trashbin items if there're these folders

Reply:
yes, no fail in CI. let's leave as it